src/mol-plugin-state/transforms/volume.ts

```diff
@@ -88,12 +88,24 @@ const VolumeFromCube = PluginStateTransform.BuiltIn({
         return {
             dataIndex,
             entryId: PD.Text(''),
+            clamp: PD.MappedStatic('off', {
```

Comment:
Wouldn't this turn off the clamp feature automatically every time we try to load a cube?